hack/externalnode/install-vm.ps1

```diff
@@ -0,0 +1,195 @@
+<#
```

Comment:
Add copyright header?

Reply:
I do not see any copyright header on any of the powershell scripts. Is this required?

Reply:
I am fine if it is expected.